process_report/process_report.py

```diff
@@ -62,15 +65,42 @@ def is_old_pi(old_pi_dict, pi, invoice_month):
     return False
 
 
+def get_invoice_bucket():
+    s3_vars = load_S3_env_vars()
+    b2_resource = boto3.resource(
+        service_name="s3",
+        endpoint_url=s3_vars["B2_ENDPOINT"],  # Backblaze endpoint
```

Comment:
These comments are all unnecessary and obvious, remove them.

Reply:
Also, why are you not reading the environment variables directly but instead passing over a dictionary?